### ISSUE TITLE
Wisp / Io tether gets leveled up along with relocate, ending up with more points in abilities than your hero level

### ISSUE THREAD

**Issue by Jorjcodeaza** [ValveSoftware/Dota2-Gameplay#2519]
### Ability name

tether and relocate 

### Description

as said before, Wisp / Io tether gets leveled up along with relocate, ending up with more points in abilities than your hero level
https://www.reddit.com/r/DotA2/comments/x1hln0/io_skill_points_bugged/ here is the reddit post with the video linked, where you can see i was level 6 and i had 7 points in abilities in total (i cannot upload the video here its too big)
basically i think the ulti searches for q allies, so it levels it up if you didnt , same goes for level 2 ulti, it makes your q level 2 if it isn't already

### Example Match ID (and possibly Timestamp)

the video in the reddit post above

**Comment by Jorjplm** [comment]
this is NOT the same bug as the one where you level up q while your attached to some1 (that was fixed)

**Comment by DarkoniusXNG** [comment]
Relocate has `"LinkedAbility"					"wisp_tether"` line. So removing this line will solve the issue.

**Comment by 667x** [comment]
Still happening 12/01/2022 
![image](https://user-images.githubusercontent.com/119571394/204994297-35a4f22a-5401-42e2-9cab-c4ee869c9370.png)

**Comment by shaileshgarg** [comment]
This has been fixed in today's update.
